src/plugins/validation/validator.js

```diff
@@ -1,5 +1,5 @@
 import PluginComponent from '../plugin.component';
-import * as validationService from './validation.service';
+import * as validationService from '@grid/core/validation/validation.service';
```

Comment:
is it more readable?:)

Reply:
No)) But I moved validation.service to core. So it is more readable than '../../core/validation/validation.service'

Reply:
Oh, i see, sorry